ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands3.java

```diff
@@ -256,17 +251,17 @@ public void testAcidMetaColumsDecode() throws Exception {
    */
   @Test
   public void testSdpoBucketed() throws Exception {
-    testSdpoBucketed(true, true, 1);
-    testSdpoBucketed(true, false, 1);
-    testSdpoBucketed(false, true, 1);
-    testSdpoBucketed(false, false,1);
-
-    testSdpoBucketed(true, true, 2);
-    testSdpoBucketed(true, false, 2);
-    testSdpoBucketed(false, true, 2);
-    testSdpoBucketed(false, false,2);
+    testSdpoBucketed(true, 1);
+    testSdpoBucketed(true, 1);
```

Comment:
There are multiple test cases with the same input parameters, so testing the same twice. Please eliminate the duplications !

Reply:
Removed the duplications.